homeassistant/components/envisalink/binary_sensor.py

```diff
@@ -58,6 +58,7 @@ def __init__(self, hass, zone_number, zone_name, zone_type, info, controller):
         """Initialize the binary_sensor."""
         self._zone_type = zone_type
         self._zone_number = zone_number
+        self._attr_unique_id = "envisalink-binary-sensor-" + str(zone_number)
```

Comment:
I guess this works, as only a single integration instance can be configured and the zone number is unique for HA for this integration because of that.

This unique ID doesn't have to be prefixed with the domain or platform.

Reply:
Sorry, Im not really a python guy and I only dabble in the actual components, I just threw this in as an example, But I assumed the same, its only a single instance and the zone number is unique in that there is only one of each of the 64 possible and not changeable by the user.  Maybe someone who is a bit more involved in the integrations could come up with a less hard coded solution.

Reply:
If you want to request a feature, please use our community forums. It has a feature request section. 👍 

If you are not planning on finishing/changing this Pull Request, please close it.

Thanks 👍 

../Frenck